WatchFaceFormat/SimpleDigital/watchface/build.gradle.kts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id("com.android.application")
+}
+
+android {
+    namespace = "com.example.simpledigital"
+    compileSdk = 33
+
+    defaultConfig {
+        applicationId = "com.example.simpledigital"
+        minSdk = 33
+        targetSdk = 33
+        versionCode = 1
+        versionName = "1.0.0"
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = true
```

Comment:
Do you want to add `isShrinkResources = false`. I'm assuming we should disable? Probably with a comment.

Reply:
I've sent a question to the watch face team. Will update on their response.

Reply:
I assume the build system won't understand that WFF references resources.